packages/@okta/vuepress-site/docs/guides/pwd-optional-overview/main/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Password optional overview
+---
+
+## Overview
+
+Most hacking-related breaches today involve weak or stolen passwords and credential security theft. Passwords are a security vulnerability because they can be easily stolen and are prone to phishing attacks. They're also not convenient either. Users often need to remember multiple passwords, change them regularly, and either memorize or store them securely somewhere.
```

Comment:
```suggestion
Most hacking-related breaches involve weak or stolen passwords and credential security theft. Passwords are a security vulnerability because they can be easily stolen and are prone to phishing attacks. They're also not convenient. Users often need to remember multiple passwords, change them regularly, and either memorize or store them securely somewhere.
```

Reply:
updated.